From 52f4b58c14a2788ca08b7d8884d586724d86fc66 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Mon, 19 Aug 2019 14:46:02 +0000
Subject: [PATCH] dnsmasq: fix build against 5.2 headers

Upstream linux y2038 work has moved some definitions SIOCGSTAMP is
defined in linux/sockios.h, not asm/sockios.h now. So we need to
add that include to fix the build.

Upstream-status: backport of http://thekelleys.org.uk/gitweb/?p=dnsmasq.git;a=commit;h=3052ce208acf602f0163166dcefb7330d537cedb

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 a/src/dnsmasq.h                   | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/tmp/work/core2-64-poky-linux/dnsmasq/2.80-r0/dnsmasq-2.80/src/dnsmasq.h b/build/tmp/work/core2-64-poky-linux/dnsmasq/2.80-r0/dnsmasq-2.80/src/dnsmasq.h
index f53e9a5be6..d9225ea473 100644
--- a/src/dnsmasq.h
+++ b/src/dnsmasq.h
@@ -138,6 +138,7 @@ typedef unsigned long long u64;
 #endif
 
 #if defined(HAVE_LINUX_NETWORK)
+#include <linux/sockios.h>
 #include <linux/capability.h>
 /* There doesn't seem to be a universally-available 
    userspace header for these. */
-- 
2.19.1

